README.md

```diff
@@ -1,3 +1,52 @@
-# Backend app
+# Music Library
 
-This app was created with `@command-shift/create-backend-app`
+## Description
+
+An API that allows users to interact with a database of artists and albums.
+
+## Purpose
+
+To improve my understanding of APIs and how to structure files on a more complex project, as well as giving me plenty of practise in using async/await Javascript and providing me with an opportunity to revist SQL.
+
+## Tech that was used
+
+- Docker: the database was set up on a Postgres server in a Docker container
+- Postgres: used both for the server mentioned above and also to manually access the databases
+- Postman: used for manually testing the API, helped with troubleshooting some errors
+
```

Comment:
You should also include the dependencies and dev dependencies. For example it's an express API that uses postgreSQL, for dev it uses mocha, chai, nodemon etc